### DISCUSSION TITLE
How to configure diff tool to not use Vim?

### DISCUSSION THREAD

**Discussion by tony-snoop** [VerifyTests/Verify#669] [Q&A] [answered]
Hi there,

Earlier versions of Verify.Xunit" (e.g. Version "17.8.1") would use a different diff tool (I think it was KDiff). 
When upgrading to the latest package - It's now using Vim! :-(

How do I configure the diff tool to revert back to what it was using earlier and not Vim?

**Comment by SimonCropp** [comment, marked as answer]
KDiff3 should resolve before Vim https://github.com/VerifyTests/DiffEngine/blob/main/docs/diff-tool.order.md

but i cant tell why it is resolving incorrectly on your machine. you will need to debug into it

**Reply by tony-snoop** [reply]
Ah-ha! Setting the `DiffEngine_ToolOrder` environment variable to `VisualStudio,KDiff3` has done the trick. Thank you @SimonCropp - much appreciated!

**Reply by SimonCropp** [reply]
still weird that vim was resolving first
